frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/review-tab.scss

```diff
@@ -5,3 +5,56 @@
 .kubevirt-create-vm-modal__review-tab-lower-section {
   margin-top: 20px;
 }
+
+.kubevirt-create-vm-modal__review-tab-section-container {
+  margin-top: var(--pf-global--spacer--md);
+}
+
+.kubevirt-create-vm-modal__review-tab-section {
+  padding-bottom: var(--pf-global--spacer--xl);
+  &__title {
+    margin-top: var(--pf-global--spacer--lg);
+    margin-bottom: var(--pf-global--spacer--md);
+  }
+  &__content {
+    // padding-left: var(--pf-global--spacer--lg);
```

Comment:
I'll take the blame for not removing this from my codepen :)

Reply:
haha... no worries. Thanks for catching it and letting me know, though. =)